mods/3d_armor/shields/crafting_guide.txt

```diff
@@ -15,7 +15,7 @@ Shields -- Crafting Guide
 [shields:shield_bronze] X = [default:bronze_ingot]
 [shields:shield_diamond] X = [default:diamond]
 [shields:shield_gold] X = [default:gold_ingot]
-[shields:shield_mithril] X = [default:mithril_ingot]
+[shields:shield_mithril] X = [moreores:mithril_ingot]
```

Comment:
Same remark as above, we shouldn't change the mod name there.

Reply:
Fixed, thanks.